Rules/CompatibilityRules/UseCompatibleSyntax.cs

```diff
@@ -29,12 +29,15 @@ public class UseCompatibleSyntax : ConfigurableRule
 
         private static readonly Version s_v6 = new Version(6,0);
 
+        private static readonly Version s_v7 = new Version(7,0);
+
         private static readonly IReadOnlyList<Version> s_targetableVersions = new []
         {
             s_v3,
             s_v4,
             s_v5,
-            s_v6
+            s_v6,
+            s_v7
```

Comment:
A comma makes the diff cleaner the next time
```suggestion
            s_v7,
```